### PR TITLE
Implement changes for second project

### PR DESCRIPTION
**Pull request by IsmailChangezi**
# Description
- Create a new column called 'Species'
- Write queries for the data
- Insert new Pokemons inside the table

## These are the snapshots
queries
Screenshot 2023-05-11 185819
Screenshot 2023-05-11 190753
Screenshot 2023-05-11 185836
Screenshot 2023-05-11 185803

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+INSERT INTO animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8 ); 
+SELECT * FROM animals;
+UPDATE animals SET neutered = false WHERE id = 1;
+INSERT INTO animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '2021-1-7' , 1, false , 15.04 ), (4, 'Devimon', '2017-5-12', 5, true,11);
+SELECT * FROM animals;
+
+-- The data for the second day's project
+
+ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+ VALUES (5, 'Charmander', '2020-2-8', 0, 'false', 11),
```
Comment (addod19) [inline data.sql]: - [ ] It would be better if we input the required weight as we have in the project requirements
Screenshot from 2023-05-11 14-34-18
Let us kindly change to `-11` instead

File: data.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ (6, 'Plantmon', '2021-11-15', 2, 'true', 5.7),
```
Comment (addod19) [inline data.sql]: - [ ] This also require `-5.7`. Here is the [link](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) to the project requirements , kindly check for the others as well

File: queries.sql [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+SELECT * FROM animals WHERE animals.weight_kg >= 10.4 AND animals.weight_kg <= 17.3;
+
+-- queries for the second day's project
+
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+SELECT * FROM animals;
```
Comment (addod19) [inline queries.sql:17]: - [ ] Since we added `species` and set it to `unspecified`, let us kindly verify it by selecting the `species`
`SELECT species from animals;`

File: queries.sql [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+ROLLBACK;
```
Comment (addod19) [inline queries.sql:19]: - [ ] Here we are required to commit after verifying the change was made. Here is how we can fix it
``COMMIT;``

File: queries.sql [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+SELECT * FROM animals;
```
Comment (addod19) [inline queries.sql:24]: - [ ] To verify that the changes persist after commit, we run `SELECT species from Animals`
Screenshot from 2023-05-11 14-41-00

File: README.md [hunk 6]
```diff
@@ -1,2 +1,234 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (addod19) [inline README.md]: - [ ] Let us kindly complete this part of our README file to make our repo more professional

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @IsmailChangezi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Correct Git-flow followed :100: 
- [ ] Queries of screenshots provided in the PR :fireworks: 
- [ ] Transactions implemented :1st_place_medal: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 7]
```diff
@@ -1,2 +1,234 @@
...
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
```
Comment (TadesseDev) [inline README.md:215]: - [ ] Dear @IsmailChangezi, As suggested by the previous reviewer, we need to make sure our readme is professional. for that, please complete these sentences and make sure you have complete information in each section.

File: README.md [hunk 8]
```diff
@@ -1,2 +1,234 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (TadesseDev) [inline README.md:233]: - [Optional] Please consider removing non-important contents from your readme, The template is designed by Microverse for us all to use to document our projects professionally and has it filled with content related to each project we may be working on. currently, I can see you have not removed the guiding contents  and `directions set by Microverser`

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @IsmailChangezi 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [x] Good job making most of the changes requested by the previous reviewer. 🙏


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 7 above)
Comment (Nemwel-Boniface) [inline README.md:215]: - [ ] This section of the README is not descriptive in any way what your visitors have to do if they like your project to show support to you. It would be better to rephrase this to include the line of text that says something like "Kindly star my project if you find it helpful" This way it will be more clear wat they have to do to show support to you.

**Review by Nemwel-Boniface — COMMENTED** [review]
##  Hello @IsmailChangezi   👋,

I am Nemwel your reviewer.


## _This is an additional review as a result of the second opinion request._

I have received your second opinion request with a complaint about the review made by the last reviewer @TadesseDev .

From my findings, the one mandatory change that was requested was very inline and was being consistent with what the first reviewer requested. Find below the specific inline comment.

## :recycle:  Required change :recycle: 

- [ ] KIndly make the requested change requested by @TadesseDev  and when you are done, feel free to request another review in your dashboard.

In case anything is not 100% clear tag me @Nemwel-Boniface so that I get the notification.

Keep rocking 🚀!

File: README.md (hunk 7 above)
Comment (IsmailChangezi) [reply README.md:215]: Hello @Nemwel-Boniface , 
I have already done that I cannot understand why you cannot see it.

**Review by Davidosky007 — APPROVED** [review]
Hi @IsmailChangezi 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* All transactions written well 👍 
* Well written PR 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
